File: src/gatling/scala/org/openapitools/client/model/FulfillmentWarehousesDTO.scala

package org.openapitools.client.model


case class FulfillmentWarehousesDTO (
    /* Список складов Маркета (FBY). */
    _warehouses: List[FulfillmentWarehouseDTO]
)
object FulfillmentWarehousesDTO {
    def toStringBody(var_warehouses: Object) =
        s"""
        | {
        | "warehouses":$var_warehouses
        | }
        """.stripMargin
}

File: ympa_fsharp_functions_server/src/model/FulfillmentWarehousesDTO.fs
namespace ympa_fsharp_functions_server.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open ympa_fsharp_functions_server.Model.FulfillmentWarehouseDTO

module FulfillmentWarehousesDTO =

  //#region FulfillmentWarehousesDTO

  [<CLIMutable>]
  type FulfillmentWarehousesDTO = {
    [<JsonProperty(PropertyName = "warehouses")>]
    Warehouses : FulfillmentWarehouseDTO[];
  }

  //#endregion
